Sebastian-Sole/handicappin
# ✨ feat: Themes

## What was done
- Updated themes and color palette (not final, but better than black and white)
- Support for dark mode
- Added theme toggle in navbar
- Changed theme used for bar chart bars

## How to test
- Visit home page
- Check that new color scheme is applied (green)
- Toggle dark mode

## Future work
- Finalize color palette and design
- Finalize navbar
